backend/config/settings.py

```
@@ -34,28 +13,14 @@
 
 TESTING = "test" in sys.argv or "pytest" in sys.modules
 
-# Patch Django template context copy for Python 3.14 compatibility
-import copy
-
-from django.template.context import BaseContext
+WS_AUTH_MIGRATION = True 
+WS_TOKEN_TIMEOUT = 3600 
 
+BASE_DIR = Path(__file__).resolve().parent.parent
```

Ruff (0.15.21)

[error] Undefined name `Path`

(F821)